internal/pkg/git/remote.go

```diff
@@ -217,7 +217,7 @@ func (r *RemoteRepository) Pull(ctx context.Context) (*PullResult, error) {
 }
 
 func (r *RemoteRepository) Free() <-chan struct{} {
-	done := make(chan struct{}, 1)
+	done := make(chan struct{})
```

Comment:
Drobnost, pri predchadzajucom PR som si vsimol, ze tu zbytocne inicializujeme `channel` s `buffer=1`,
kedze do `ch` nezapisujeme, iba zavolanim `close(done)` dame vediet, ze je operacia dokoncena.

(znie to divne, ze do `ch` vobec nezapisujeme, ale je to bezne, vid napr. podobny kod tu: https://www.reddit.com/r/golang/comments/utafpe/understanding_the_done_channel_pattern/)

-----

Je to ina situacia ako tu (co sme riesili nedavno), ze do `ch` zapisujeme prave jednu hodnotu, a nechceme aby bol zapis blokovany, ak ta hodnota nikoho nezaujima.

https://github.com/keboola/keboola-as-code/blob/f448c6c4e2cdb767a2b11235b4afe9e8d8483cb8/internal/pkg/template/repository/manager/manager.go#L125-L166